frontend/src/component/project/Project/ProjectInfo/ToggleTypesWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import { IFeatureToggleListItem } from '../../../../interfaces/featureToggle';
+import { useMemo } from 'react';
+import {
+    StyledCount,
+    StyledDivInfoContainer,
+    StyledParagraphGridRow,
+    StyledParagraphSubtitle,
+} from './ProjectInfo.styles';
+import { getFeatureTypeIcons } from '../../../../utils/getFeatureTypeIcons';
+
+export interface IToggleTypesWidgetProps {
+    features: IFeatureToggleListItem[];
+}
+
+export const ToggleTypesWidget = ({ features }: IToggleTypesWidgetProps) => {
+    const { release, experiment, operational, kill, permission } =
+        useMemo(() => {
+            const release =
+                features?.filter(feature => feature.type === 'release')
+                    .length || 0;
+            const experiment =
+                features?.filter(feature => feature.type === 'experiment')
+                    .length || 0;
+            const operational =
+                features?.filter(feature => feature.type === 'operational')
+                    .length || 0;
+            const kill =
+                features?.filter(feature => feature.type === 'kill-switch')
+                    .length || 0;
+            const permission =
+                features?.filter(feature => feature.type === 'permission')
+                    .length || 0;
+
+            return {
+                release,
+                experiment,
+                operational,
+                kill,
+                permission,
+            };
+        }, [features]);
```

Comment:
this is a potential bug. react hooks should either depend on primitive types or use JSON.stringify in deps array

Reply:
Didn't know that, thanks